charts/activiti-cloud-full-example/Makefile

```diff
@@ -3,13 +3,15 @@ VERSION := $(or ${VERSION}, $(shell cat ../../VERSION))
 PREVIEW_NAMESPACE_LOWERCASE := $(shell echo ${PREVIEW_NAME} | tr "[:upper:]" "[:lower:]")
 GITHUB_CHARTS_BRANCH := $(or ${GITHUB_CHARTS_BRANCH}, gh-pages)
 GITHUB_CHARTS_DIR := activiti-cloud-helm-charts
+CLIENT_SECRET := $(or ${CLIENT_SECRET}, dry-run)
```

Comment:
can we change to KEYCLOAK_CLIENT_SECRET to be more speaking?

Reply:
Done